### ISSUE TITLE
DDPG on MountainCar

### ISSUE THREAD

**Issue by gearsuccess** [p-christ/Deep-Reinforcement-Learning-Algorithms-with-PyTorch#38]
Hi p-christ,

Thanks for this amazing contribution. Recently, I tried the implementation of DDPG for MountainCar (with default parameters in results/Mountain_Car.py). However, the results are quite different from the plot in READ.me. Could you plz give me some advices?

Thanks

**Comment by chrisdcs** [comment]
Same question. 

And TD3 results are not different as well.

**Comment by Ritam9** [comment]
Indeed. There is a tremendous difference in the results, in PPO as well.
